sandbox/technical_user_guide.md

```diff
@@ -136,17 +131,57 @@ Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOmY5NzgwZDMyMzU4OGYxY2RmYz
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-**cURL command**
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which performs Basic authentication by passing in an additional header**
```

Comment:
Maybe: "Option 1, which requires that base-64 encoding be performed on the clientId and secret"

Reply:
Accepted, resolved.